behat/tests/bootstrap/v2/reporting/sections/AccountsSectionTrait.php

```diff
@@ -319,24 +319,24 @@ public function iShouldSeeTheExpectedAccountsOnSummaryPage()
             $headHtml = trim(strtolower($tableHeader->getHtml()));
             assert(
                 str_contains($headHtml, $this->accountList[$tRowKey]['accountType']),
-                sprintf('matching account %s ', $this->accountList[$tRowKey]['accountType'])
+                $this->formatAssertResponse($this->accountList[$tRowKey]['accountType'], 'Not found', 'Accounts Type', $this->getCurrentUrl())
```

Comment:
I wonder if we could wrap assert itself in a helper function that does all of this? We have the same value passed to `str_contains()` and `formatAssertResponse()` and $this->getCurrentUrl() could just be called in the function rather than passed as an argument leaving you with:

- HTML to match against,
- Value to match against
- Found (or not found) value (could leave this as a null field at the end so null values default to 'Not found')
- Name of element matching against ($comparisonSubject)

Reply:
Ok I've done this. It does make it slightly less flexible but it looks a bit neater.